mathext/internal/cephes/psi.go

```diff
@@ -0,0 +1,146 @@
+// Copyright ©2025 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+/*
+Cephes Math Library Release 2.8:  June, 2000
+Copyright 1984, 1987, 1992, 2000 by Stephen L. Moshier
+*/
+
+package cephes
+
+import "math"
+
+var A []float64 = []float64{
```

Comment:
Does this need to be exported?

Reply:
This is not needed, sorry for accidentally exporting it. This is made private in the latest commit.